src/main/java/com/example/demo/controller/TeamController.java

```diff
@@ -0,0 +1,30 @@
+package com.example.demo.controller;
+
+import com.example.demo.models.Team;
+import com.example.demo.service.TeamService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/team")
+@AllArgsConstructor
+public class TeamController {
+    private final TeamService teamService;
+
+     @PostMapping
+     public ResponseEntity<Team> addTeam(@RequestBody Team team){
```

Comment:
Try to keep your code in one style. If you use ResponseEntity here - you probably should use it in other controllers